examples/remoteBuild.md

```diff
@@ -48,7 +48,7 @@ one pod in the output:
 
 <!-- @remoteOverlayBuild @testAgainstLatestRelease -->
 ```
-target="https://github.com/kubernetes-sigs/kustomize//examples/multibases/dev/?ref=v1.0.6"
+target="https://github.com/kubernetes-sigs/kustomize//examples/multibases/dev/?timeout=120&ref=kustomize/v5.1.1"
```

Comment:
When I run this, the timeout is too small for me. I had to up the timeout to 3 minutes on my machine, so I'm not sure that this timeout will be universally ok for all users. I believe the reason that v5 takes so much longer to clone than v1 is because the repo is much, much bigger now. 

I think it will be very confusing for users (and a bad user experience) who run this command and see it hang for over a minute while kustomize tries to clone the large repo. I myself thought it was broken at first and that there must be something wrong with the syntax or example until it timed out at 2 minutes.

I see in a previous thread that you had it originally using `v3.3.1`. When I run that, it returns instantly and without printing the confusing warning. I think that will be the best user experience for users who follow these docs.

Really sorry for the back and forth (I know it's annoying) but I think it would be best to stick with v3.3.1 for this (fyi @koba1t).